symbols/icled_up.ts

```diff
@@ -0,0 +1,27 @@
+import { rotateSymbol } from "drawing/rotateSymbol"
+import icled_right from "./icled_right"
+
+const baseSymbol = rotateSymbol(icled_right, "up")
+
+// Flip path25-0 and path78 over X axis
```

Comment:
use a utility function for flipping from `drawing/`, if it doesn't exist you should create it. It's not maintainable like this because it's so custom.

I'm also pretty sure it's not correct w.r.t. the pins

Reply:
oh it does work because of the rotation with the pins, but it's crazy hacky